src/test/java/seedu/address/logic/ModelStub.java

```diff
@@ -0,0 +1,381 @@
+
+    @Override
+    public boolean hasEvent(Event event) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void deleteEvent(Event target) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void addEvent(Event event) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void setEvent(Event target, Event editedEvent) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public ObservableList<Event> getFilteredEventList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void updateFilteredEventList(Predicate<Event> predicate) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void commitEventList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void undoEventList() throws VersionedListException {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void redoEventList() throws VersionedListException {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public ObservableList<Contact> getSortedContactList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void updateSortedContactList(Comparator<Contact> comparator) {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void commitModuleList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void redoModuleList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void undoModuleList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void commitContactList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void redoContactList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void undoContactList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void commitTodoList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void redoTodoList() {
+        throw new AssertionError("This method should not be called.");
+    }
+
+    @Override
+    public void undoTodoList() {
+        throw new AssertionError("This method should not be called.");
+    }
+    @Override
+    public void commit(int i) {
+        throw new AssertionError("This method should not be called.");
+    }
```

Comment:
Should separate these methods with a line.

Reply:
Ok